test-clients/yahcli/assets/screened-launch.sh

```diff
@@ -2,7 +2,5 @@
 java -jar /opt/bin/yahcli.jar "$@" 2>syserr.log \
   | grep -v sun.reflect.Reflection.getCallerClass
 RC=$?
-if [ ! $RC -eq 0 ]; then
```

Comment:
Why remove this?

Reply:
Answered out of band. `$RC` is always 0, so the error log was never printed.

Reply:
we discuss it with 
@tinker-michaelj 
 that we will print the log error in case no errors it will print empty otherwise it will show the error